test_resources/log4j.properties

```diff
@@ -15,6 +15,7 @@ log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.ConversionPattern=%d [%t] %-5p%c - %m%n
 
 # customizations to logging by package
+log4j.logger.org.quartz=ERROR
```

Comment:
isn't the default log level already `ERROR`?

Reply:
Right, I'll get rid of this